Sources/SWBTestSupport/RunDestinationTestSupport.swift

```diff
@@ -326,7 +335,12 @@ extension RunDestinationInfo {
         }
         switch imageFormat(core) {
         case .elf:
-            environment.prependPath(key: "LD_LIBRARY_PATH", value: toolchain.path.join("usr/lib/swift/\(platform)").str)
+            if case let .appleSDK(platform: platform, _, _) = buildTarget {
+                environment.prependPath(key: "LD_LIBRARY_PATH", value: toolchain.path.join("usr/lib/swift/\(platform)").str)
+            } else {
+                // Fall back to the OS provided Swift runtime
```

Comment:
If it's a Swift SDK you still need to specify this env var. You can parse the triple using LLVMTriple and use the 'os' component as the platform.